Make-a-pull-request.md

```diff
@@ -130,6 +132,47 @@ Notice that in this example:
 * No proof of correctness was needed, but this might not be the case for your PR.
 * The CI checks were failing as soon as the PR was opened. If you see this, you should investigate.
 
+### Pull request lifecycle
+```mermaid
```

Comment:
Please add a blank line above the flowchart to make the source file easier to read

Reply:
Nice, that makes it much clearer to read.